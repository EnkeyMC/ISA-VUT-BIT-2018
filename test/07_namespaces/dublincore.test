NAME=Dublin Core namespace test
ARGS=http://www.mocky.io/v2/5beef6e22e0000c836eeeb4b -a -T -u
RC=0
CHECK_OUTPUT=1
