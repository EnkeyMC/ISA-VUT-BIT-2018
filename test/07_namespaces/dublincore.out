*** XML.com ***
Processing Inclusions with XSLT
URL: http://xml.com/pub/2000/08/09/xslt/xslt.html
Aktualizace: 
Autor: 

Putting RDF to Work
URL: http://xml.com/pub/2000/08/09/rdfdb/index.html
Aktualizace: 2000-01-01T12:00+00:00
Autor: Rael Dornfest (mailto:rael@oreilly.com)
